### PR TITLE
NEW date and user signature on proposal (Issue 16062)

### PR DESCRIPTION
**Pull request by lvessiller-opendsi**
NEW date and user signature on proposal (Issue 16062)
- add "date_signature" and "fk_user_signature" fields in "propal" table and in class

### CODE REVIEW

**Comment by BB2A-Anthony** [comment]
Is good proposal
date_signature is not éditable in formconfirm?
date_signature is dates user confirm signed in dolibarr?
If user confirm also days after signted real

**Comment by lvessiller-opendsi** [comment]
@bb2a 
It's a feature and I used horodate such as validation and close date (not modified in propal card).
I understand what you want to do, but I already have a lot of changes.
I think it could be a new feature if we need to edit signature date directly from propal card.

File: htdocs/comm/propal/card.php [hunk 1]
```diff
@@ -642,7 +642,7 @@
 			if ($object->statut == $object::STATUS_VALIDATED) {
 				$db->begin();
 
-				$result = $object->cloture($user, GETPOST('statut', 'int'), GETPOST('note_private', 'restricthtml'));
+				$result = $object->signature($user, GETPOST('statut', 'int'), GETPOST('note_private', 'restricthtml'));
```
Comment (eldy) [inline htdocs/comm/propal/card.php:645]: Currently the user that set the status of proposal to "signed" by customer is saved into field   fk_cloture (for "closed as signed").
What will be the goal of having a second field ?
Is it to track a status signed by customer or signed by the company ?
Is it to add a second level of validation of proposal by a second user different than the one who validate the proposal  before sending it to the customer ?

Comment (lvessiller-opendsi) [reply htdocs/comm/propal/card.php:645]: @eldy
Having a second field is needed to differenciate user who sign and user who close the commercial proposal (as we defined in issue 16062).

Comment (eldy) [reply htdocs/comm/propal/card.php:645]: I see. Thanks.

**Comment by lvessiller-opendsi** [comment]
@eldy
I tried to resolve conflicts.
I got a Travis error but it should not concern this feature.

**Comment by BB2A-Anthony** [comment]
No translate in en_US ?

**Comment by lvessiller-opendsi** [comment]
@bb2a 
> No translate in en_US ?

Completed in new PR 17056

**Comment by BB2A-Anthony** [comment]
Ok sorry

**Pull request merged**